src/Microsoft.AspNetCore.Authentication.OAuth/OAuthHandler.cs

```diff
@@ -169,7 +166,12 @@ public OAuthHandler(HttpClient backchannel)
                 throw new ArgumentNullException(nameof(context));
             }
 
-            var properties = new AuthenticationProperties(context.Properties);
+            var properties = new AuthenticationProperties(context.Properties)
+            {
+                // Review: should we add an option to configure the state lifetime?
+                ExpiresUtc = DateTimeOffset.UtcNow + TimeSpan.FromHours(1)
```

Comment:
Updated to use a 15mn lifetime (I just rememberd the OIDC options explicitly replace the default lifetime :smile:)